packages/api-browser/src/screen-snippet.js

```diff
@@ -1,4 +1,5 @@
 /* globals html2canvas */
+import 'html2canvas';
```

Comment:
Could we not do `import html2canvas from 'html2canvas'` here? Then we could remove the eslint comment?